llvm/lib/Target/AMDGPU/AMDGPURegisterBankInfo.cpp

```diff
@@ -3355,6 +3429,28 @@ AMDGPURegisterBankInfo::getDefaultMappingVOP(const MachineInstr &MI) const {
                                MI.getNumOperands());
 }
 
+const RegisterBankInfo::InstructionMapping &
+AMDGPURegisterBankInfo::getDefaultMappingVOPWithPreassignedDef(
+    const MachineInstr &MI) const {
+  SmallVector<const ValueMapping *, 8> OpdsMapping(MI.getNumOperands());
+  const MachineRegisterInfo &MRI = MI.getMF()->getRegInfo();
+  // Dst reg bank should have been set already by uniformity info
+  OpdsMapping[0] =
+      getPreAssignedOpMapping(MI.getOperand(0).getReg(), MRI, *TRI);
+
+  for (unsigned i = 1, e = MI.getNumOperands(); i != e; ++i) {
```

Comment:
`i` -> `I`, `e` -> `E`

Reply:
I see both lower case and upper case `i` and `I` used equally often. Is there a particular reason why to use upper case `I`?

Reply:
Where in the codebase do you see `i` ? A quick search for `i =` shows no result for me
In any case, coding style says variables should be uppercase so I think it's better to follow the coding style

Reply:
From /llvm/lib/
`unsigned I =` 1185 `int I =` 276
`unsigned i =` 2833 `int i =` 551

Reply:
Ah right, sure, but I still think that the coding style should be followed: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly
So `I` is technically correct (the best kind of correct :) )

OTOH, this is really a microscopic issue (after all this is just a loop count) and I'm not going to reject the PR over that so do whatever you prefer in the end